pkg/app/server_test.go

```diff
@@ -334,6 +336,7 @@ kube_pod_info{namespace="default",pod="pod0",uid="abc-0",host_ip="1.1.1.1",pod_i
 kube_pod_labels{namespace="default",pod="pod0",uid="abc-0"} 1
 kube_pod_owner{namespace="default",pod="pod0",uid="abc-0",owner_kind="",owner_name="",owner_is_controller=""} 1
 kube_pod_restart_policy{namespace="default",pod="pod0",uid="abc-0",type="Always"} 1
+kube_pod_service_account{namespace="default",pod="pod0",uid="abc-0",service_account=""} 1
```

Comment:
QQ: can this metric kube_pod_service_account be used in alerts? Or only informational purpose.

Reply:
@CatherineF-dev my use case is to have this for for informational purposes so our users can query metrics by their service account. All pods have the service account in the podspec according to [the docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server) so I think it could be used for alerts. However if it's a question about cardinality, I'm not too sure.